test/test_library.py

```diff
@@ -708,13 +708,13 @@ def test_if_def_false_complete(self):
         self._assert_dest(b"/base/not_played")
 
     def test_first(self):
```

Comment:
Could you try and test this `%first{}` function with the new `genres` field, to show how the user may use it to extract the first genre?

Reply:
Currently the only way to use `%first` with these `MULTI_VALUE_DSV` fields is [this](https://github.com/beetbox/beets/pull/4743#issuecomment-2014092144) (for the record, that doesn't even work for me and I have to do `%first{%first{$albumartists,1,0,␀},1,0,\\}` in my own config) which I think is pretty awful, not documented anywhere and not even confirmed as the intended method. 

Without it being confirmed that that is the intended method of use, or a future improvement to the ergonomics, I wouldn't want to encode this in a test as the expected usage. This PR just brings up the genres field to the same state as the artists field, and any work beyond that is out of scope.